libs/langchain/langchain/schema/runnable/passthrough.py

```diff
@@ -45,6 +47,26 @@ class RunnablePassthrough(RunnableSerializable[Input, Input]):
 
     input_type: Optional[Type[Input]] = None
 
+    func: Optional[Callable[[Input], None]] = None
```

Comment:
if this is only meant to be used for callbacks could we make var name indicate that (eg callback_func)?

Reply:
fair